website/index.html

```diff
@@ -193,9 +193,9 @@
           <div class="offset-md-2 col-md-8 ">
             <div class="row">
               <div class="col-12 header-text wrapped-box">
-                <p class="text-left"><span>April 4-5</span></p>
-                <p class="text-left"><span>Penn State University</span></p>
-                <p class="text-left"><span>Business Building</span></p>
+                <p class="text-left"><span>HackPSU Has Been Canceled!</span></p>
```

Comment:
I'm not sure if the exclamation point fits, could you consider removing it?

Reply:
done